content/en/containers/troubleshooting/log-collection.md

```diff
@@ -0,0 +1,688 @@
+
+Note that the directory relative to the underlying host may be different due to specific configuration of the Docker daemon—this is not an issue pending a correct Docker volume mapping. For example, use `-v /data/docker/containers:/var/lib/docker/containers:ro` if the Docker data directory has been relocated to `/data/docker` on the underlying host.
+
+If logs are collected but single lines appear to be split, check that the Docker daemon is using the [JSON logging driver](#different-docker-log-driver).
+
+### Host based Agent
+
+If you are installing the Agent on the host as opposed to running in a Docker container, the user `dd-agent` needs to be added to the Docker group to have permission to read from the Docker socket. If you see the following error logs from the agent:
+
+```text
+<TIMESTAMP> UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:360 in initListenerCandidates) | docker listener cannot start, will retry: temporary failure in dockerutil, will retry later: could not determine docker server API version: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/version: dial unix /var/run/docker.sock: connect: permission denied
+<TIMESTAMP> UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:123 in collect) | Unable to collect configurations from provider docker: temporary failure in dockerutil, will retry later: could not determine docker server API version: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/version: dial unix /var/run/docker.sock: connect: permission denied
+```
+
+Add the Agent to the Docker user group, perform the following command:
+```
+usermod -a -G docker dd-agent
+```
+**Note:** When you install the Agent on the host, the Agent does not have permission to access` /var/lib/docker/containers` as these require root access. As a result it will collect logs from the Docker socket.
+
+
+### Different Docker log driver
+
+Docker’s default is the [json-file logging driver][23] so the Agent tries to read from this structure first. If your containers are set to use a different logging driver, the Logs Agent will indicate that it is able to successfully find your containers but it isn’t able to collect their logs from the file. In Docker environments, Datadog recommends using the `json-file` log driver for the optimal Agent experience. However, the Agent can also be configured to read from the `journald` logging driver.
+
+1. If you’re unsure of which logging driver your containers are using, use `docker inspect <CONTAINER_NAME>` to see what logging driver you have set. The following block appears in the Docker Inspect when the container is using the JSON logging driver
+
+```
+"LogConfig": {
+    "Type": "json-file",
+    "Config": {}
+},
+```
```

Comment:
This is to line it up under the numbered steps

````suggestion
   ```
   "LogConfig": {
       "Type": "json-file",
       "Config": {}
   },
   ```
````